modules/browsiRtdProvider.js

```diff
@@ -57,7 +57,7 @@ export function addBrowsiTag(data) {
   script.setAttribute('prebidbpt', 'true');
   script.setAttribute('id', 'browsi-tag');
   script.setAttribute('src', data.u);
-  script.prebidData = deepClone(data);
+  script.prebidData = deepClone(typeof data === 'string' ? Object(data) : data)
```

Comment:
Version 1 of just-clone casts everything to object, even though in this case it was a string.

The newer versions don't do that, just like klona.

This is here to preserve that behavior.